satellite/mailservice/service.go

```diff
@@ -66,10 +69,39 @@ func New(log *zap.Logger, sender Sender, templatePath string) (*Service, error)
 	//	return nil, err
 	//}
 
-	service.html, err = htmltemplate.ParseGlob(filepath.Join(templatePath, "*.html"))
+	//service.html, err = htmltemplate.ParseGlob(filepath.Join(templatePath, "*.html"))
+	r, err := zip.OpenReader("/tmp/app/assets.zip")
```

Comment:
This needs to know where the static assets zip is located. It probably needs a config option.